plexpy/webserve.py

```diff
@@ -4504,7 +4505,7 @@ def get_item_watch_time_stats(self, rating_key=None, grouping=None, query_days=N
 
         if rating_key:
             item_data = datafactory.DataFactory()
-            result = item_data.get_watch_time_stats(rating_key=rating_key, grouping=grouping,
+            result = item_data.get_watch_time_stats(rating_key=rating_key, media_type=media_type, grouping=grouping,
```

Comment:
Line break so it's cleaner.
```suggestion
            result = item_data.get_watch_time_stats(rating_key=rating_key,
                                                    media_type=media_type,
                                                    grouping=grouping,
```